index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <title>«PINK»</title>
+</head>
+
+<body class="page__body">
+  <header class="main-header">
+    <a class="main-header__logo">
+      <img src="#" width="74" height="23" alt="Логотип Pink">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Фотографии
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Конкурс
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <div class="header-main">
+      <p class="header-slogan">Взгляните на жизнь иначе!</p>
+      <a class="header-main__img">
+        <img src="#" width="320" height="286" alt="Рука с телефоном">
+      </a>
+    </div>
+    <div class="header-footer">
+      <a class="button" href="#">Скачать приложение</a>
+      <div class="hader-footer__download">
```

Comment:
Ну здесь банальная опечатка.
```suggestion
      <div class="header-footer__download">
```